packages/@n8n/chat/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@n8n/chat",
-  "version": "0.4.1",
+  "version": "0.5.1",
```

Comment:
version for all packages are managed via the release process. we should be able to make manual releases with `-alpha` suffixes, but we should not be upgrading the versions in the repo.
But if that process doesn't work for this package, perhaps we should move this out into a separate repo?

Reply:
I don't have a strong opinion about this. We're planning to replace the built-in chat with with components from the `@n8n/chat` package, and having it separately would probably make it a bit harder. I don't expect the components to change too often, though.

Reply:
btw, why are we publishing this package? is it not possible to use the local build of this in dev mode?